cg/meta/compress/compress.py

```diff
@@ -275,27 +280,37 @@ def get_all_non_fastq_tags(fastq_file: File) -> List[str]:
         fastq_tags.remove(SequencingFileTag.FASTQ)
         return fastq_tags
 
-    def add_fastq_hk(self, sample_obj: Sample, fastq_first: Path, fastq_second: Path) -> None:
-        """Add FASTQ files to Housekeeper."""
+    def add_fastq_hk(
+        self,
+        spring_path: Path,
+        sample_internal_id: str,
+        fastq_first: Path,
+        fastq_second: Path,
+    ) -> None:
+        """Add decompressed FASTQ files to Housekeeper."""
+        fastq_tags: List[str] = self.get_fastq_tag_names_from_spring_path(spring_path)
 
-        if not sample_obj.application_version.application.is_external:
-            flow_cell_id: str = self.get_flow_cell_id(fastq_path=fastq_first)
-            fastq_tags: List[str] = [flow_cell_id, SequencingFileTag.FASTQ]
-        else:
-            fastq_tags: List[str] = [sample_obj.internal_id, SequencingFileTag.FASTQ]
         LOG.info(
-            f"Adds {fastq_first}, {fastq_second} to bundle {sample_obj.internal_id} with tags {fastq_tags}"
+            f"Adds {fastq_first}, {fastq_second} to bundle {sample_internal_id} with tags {fastq_tags}"
         )
         if self.dry_run:
             return
 
         LOG.info("Updating files in Housekeeper...")
         for fastq in [fastq_first, fastq_second]:
             self.hk_api.add_and_include_file_to_latest_version(
-                bundle_name=sample_obj.internal_id, file=fastq, tags=fastq_tags
+                bundle_name=sample_internal_id, file=fastq, tags=fastq_tags
             )
         self.hk_api.commit()
 
+    def get_fastq_tag_names_from_spring_path(self, spring_path: Path) -> List[str]:
```

Comment:
Maybe add one or some unit tests for this new function?

Reply:
Added a test now, have a look! @diitaz93

Reply:
Great 💯